docs/chapters/cross-multi-modality/index.md

```diff
@@ -0,0 +1,42 @@
+## Multi & Cross Modality
+
+Jina is a data type-agnostic framework. It basically can work with any kind of data. 
+
+Jina does not only offer the possibility to work with any data, but also allows you to run cross- and multi-modal search flows. 
+To better understand what this implies we first need to understand the concept of modality. 
+
+One may think that different modalities correspond to different kinds of data (images and text in this case).
+However, this is not accurate. For example, one can do cross-modal search by searching images from different points of view, 
```

Comment:
"A misconception is that modalities correspond to different kinds of data, such as images or text. However, one can also do a cross-modal search of images from images of different points of view or paragraphs of text from matching titles. Therefore, one can think of a modality as a given data distribution of the input."

Reply:
Let's keep the wording simple and second-person style

- `one` sounds like we're royalty. `you` or `we` is much more direct and commonly used irl
- `points of view` -> `perspectives` (I saw points of view at first and thought opinions)